ffv1.md

```diff
@@ -1124,15 +1124,15 @@ If state_transition_delta is not present in the FFV1 bitstream, all Range coder
 \* Encoders MUST NOT store bits_per_raw_sample = 0
 Decoders SHOULD accept and interpret bits_per_raw_sample = 0 as 8.
 
-### h_chroma_subsample
+### log2_h_chroma_subsample
 
-PDF:`h_chroma_subsample` indicates the subsample factor between luma and chroma width ($chroma\_width=2^{-log2\_h\_chroma\_subsample}luma\_width$).
-RFC:`h_chroma_subsample` indicates the subsample factor between luma and chroma width (`chroma_width = 2^(-log2_h_chroma_subsample) * luma_width`).
+PDF:`log2_h_chroma_subsample` indicates the subsample factor, stored in powers to which the number 2 must be raised, between luma and chroma width ($chroma\_width=2^{-log2\_h\_chroma\_subsample}luma\_width$).
```

Comment:
I suggest avoiding lowercase `must` to not confuse with RFC2119 meaning. Also `stored in powers to which the number 2 must be raised` seems passive wording. Could you say `the base-two logarithm of the subsample factor`...?

Reply:
same comment applies 3x below

Reply:
PR updated with the suggested wording.